cmd/kubepkg/cmd/root.go

```diff
@@ -156,3 +157,15 @@ func (r *rootOptions) validate() error {
 
 	return nil
 }
+
+func run(ro *rootOptions, buildType kubepkg.BuildType) error {
+	client := kubepkg.New()
+	builds, err := client.ConstructBuilds(
+		buildType, ro.packages, ro.channels, ro.kubeVersion, ro.revision,
+		ro.cniVersion, ro.criToolsVersion, ro.templateDir,
+	)
+	if err != nil {
+		return errors.Wrap(err, "running kubepkg")
+	}
+	return client.WalkBuilds(builds, ro.architectures, ro.specOnly)
+}
```

Comment:
From a general usage I'm wondering if something like this would be even better:
```go
func run(ro *rootOptions, buildType kubepkg.BuildType) error {
	client := kubepkg.New()
	builds, err := client.ConstructBuilds(ro)
	if err != nil {
		return err
	}
	return build.Walk()
}
```

Reply:
@saschagrunert -- :point_up_2: looks much cleaner :)

Reply:
I can change it as a follow-up if you want. I wanted to bump the test coverage in any case for that package after that PR got merged.

Reply:
Yep. Follow up is fine.